Add GitHub Actions Workflow

byRoadrunner opened the pull request:
This pull request includes:

- [ ] breaking changes
- [x] no breaking changes

### Changes made to the repository:

- Added GitHub Actions Workflow for build and artifact upload


### Documentation of test results:

- https://github.com/CloudNetService/website/actions/runs/790196018
- GitHub Actions

byRoadrunner committed:
Added GitHub Actions Support

byRoadrunner committed:
Changed artifact name to use context instead of environment variable

Sarsum commented on .github/workflows/yarn.yml:
```diff
@@ -0,0 +1,31 @@
+        uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: website-node${{ matrix.node }}-${{ github.sha }}
```
Please set the artifact name to something like "cloudnetservice-website..."

Sarsum reviewed (changes requested):
Otherwise everything looks good, thank you!

byRoadrunner committed:
Changed artifact name

As requested in https://github.com/CloudNetService/website/pull/7#discussion_r628745308
